experimental/msal-react/src/index.ts

```diff
@@ -0,0 +1,22 @@
+// TODO: How do we get the token before executing an API call in pure TypeScript, outside of a React component context?
```

Comment:
@AndrewCraswell Can you clarify this please?

Reply:
Clarified in comment below